src/Containers/App/App.js

```diff
@@ -1,15 +1,15 @@
 import React, { Component } from "react"
 import { NavLink, Route, Switch, withRouter } from "react-router-dom"
 import NotesSection from "../NotesSection/NotesSection"
-import EditNoteForm from "../EditNoteForm/EditNoteForm"
 import NoteForm from "../NoteForm/NoteForm"
 import NotFound from "../../Components/NotFound/NotFound"
 import { connect } from "react-redux"
 import { setNotes, setLoading, setError } from "../../actions"
 import API from '../../utils/api';
 
 class App extends Component {
-  async componentDidMount() {
+  
+  getNotes = async () => {
     const { setNotes, setLoading } = this.props
     try {
       const notes = await API.fetchData('notes', 'GET');
```

Comment:
I created a thunk to do this functionality, so then for the note form we could dispatch that thunk again instead of passing it as a prop.

Reply:
Yes, exactly what I was thinking when I saw your thunk!